### ISSUE TITLE
Why do I need to load 2D data when training 3D data？

### ISSUE THREAD

**Issue by APeiZou** [mks0601/3DMPPE_POSENET_RELEASE#95]
![image](https://user-images.githubusercontent.com/49524647/123754058-0c4aa780-d8ed-11eb-85a0-29b875ce6406.png)
Why do I need to load 2D MPII dataset when training the 3D Human36M dataset？

**Comment by mks0601** [comment]
The model is jointly trained on 2D MPII and 3D H36M datasets.

**Comment by APeiZou** [comment]
@mks0601   They  are   have   deffierent    num_keypoints?  how many  outputs  numkeypoints?

**Comment by mks0601** [comment]
The joints are transformed to those of 3D datasets (https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/3f92ebaef214a0eb1574b7265e836456fbf3508a/data/dataset.py#L105)

**Comment by APeiZou** [comment]
@mks0601 Can I only  train in  Human36 datasets ?  How to init_weight of the model when training? Do I need to load the pretrained model?

**Comment by mks0601** [comment]
You can teain only on h36m by removing mpii from the traingset. You dont have to load it.

**Comment by APeiZou** [comment]
The values of vis are all zero when  I load Human36 dataset，and the loss is zero。
![image](https://user-images.githubusercontent.com/49524647/123779423-bda90780-d904-11eb-86ce-94fee64d8c3f.png)

**Comment by mks0601** [comment]
Please give me full descriptions of your situation. I cannot give you an answer from a screenshot.

**Comment by APeiZou** [comment]
![image](https://user-images.githubusercontent.com/49524647/123780374-b6cec480-d905-11eb-89c2-e5c2559d0127.png)
The vis Attributes of target is zero  and  the loss are always zero。

**Comment by mks0601** [comment]
joint_vis is set in [here](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/3f92ebaef214a0eb1574b7265e836456fbf3508a/data/dataset.py#L74). The original codes work well, so you'd better check your modifications.

**Comment by APeiZou** [comment]
@mks0601   I have not bbox_root_human36m_output.json  file  ，can I train the model in Human36 datasets ?
![image](https://user-images.githubusercontent.com/49524647/123784711-4b3b2600-d90a-11eb-9f3b-985ec7a37e53.png)

**Comment by mks0601** [comment]
You can downlaod from here.
https://github.com/mks0601/3DMPPE_POSENET_RELEASE#results

**Comment by st20042066** [comment]
@APeiZou Hi, did you fix the problem about the vis Attributes of target is all zero? I think I also have this problem and I don't know what's wrong. Can you give me some advice please? Thank you so much.

**Comment by st20042066** [comment]
![螢幕擷取畫面 2024-02-22 112353](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/62920600/3f37e3b6-da09-4ba2-9212-65aed8d59df4)
@mks0601 I print the conditional parameter and find out that the data has huge error, Can anyone help me?
